examples/lightning_base.py

```diff
@@ -96,25 +93,14 @@ def optimizer_step(self, epoch, batch_idx, optimizer, optimizer_idx, second_orde
         optimizer.zero_grad()
         self.lr_scheduler.step()
```

Comment:
Do we still need to manually call `self.lr_scheduler` or can we have lightning handle this for us?